lib/cluster/base.rb

```diff
@@ -100,6 +100,7 @@ def self.service_role_policy_document
                 "ec2:*",
                 "iam:PassRole",
                 "cloudwatch:*",
+                "logs:*",
```

Comment:
The changes in this file are necessary to allow the cluster's service role to create VPC flow logs